libs/blocks/aside/aside.css

```diff
@@ -153,9 +207,24 @@
 
 .aside.split .foreground.container {
   width: 100%;
+  max-width: 100%;
+  flex-direction: column;
+  z-index: 1;
 }
 
 .aside.split .foreground.container .text {
+  padding: 40px 0;
```

Comment:
Can we use spacing vars here? `--spacing-l`

Reply:
done